bin/i18n/i18n_script_utils.rb

```diff
@@ -12,59 +12,20 @@
 
-CROWDIN_PROJECTS = {
-  codeorg: {
-    config_file:            CDO.dir('bin/i18n/crowdin/config/codeorg_crowdin.yml'),
-    identity_file:          CDO.dir('bin/i18n/crowdin_credentials.yml'),
-    etags_json:             CDO.dir('bin/i18n/crowdin/etags/codeorg_etags.json'),
-  },
-  'codeorg-markdown': {
-    config_file:            CDO.dir('bin/i18n/crowdin/config/codeorg_markdown_crowdin.yml'),
-    identity_file:          CDO.dir('bin/i18n/crowdin_credentials.yml'),
-    etags_json:             CDO.dir('bin/i18n/crowdin/etags/codeorg-markdown_etags.json'),
-  },
-  'hour-of-code': {
-    config_file:            CDO.dir('bin/i18n/crowdin/config/hourofcode_crowdin.yml'),
-    identity_file:          CDO.dir('bin/i18n/crowdin_credentials.yml'),
-    etags_json:             CDO.dir('bin/i18n/crowdin/etags/hour-of-code_etags.json'),
-  },
-  'codeorg-restricted': {
-    config_file:            CDO.dir('bin/i18n/crowdin/config/codeorg_restricted_crowdin.yml'),
-    identity_file:          CDO.dir('bin/i18n/crowdin_credentials.yml'),
-    etags_json:             CDO.dir('bin/i18n/crowdin/etags/codeorg-restricted_etags.json'),
-  },
-}.freeze
-
-CROWDIN_TEST_PROJECTS = {
-  'codeorg-testing': {
-    config_file:            CDO.dir('bin/i18n/crowdin/config/codeorg-testing_crowdin.yml'),
-    identity_file:          CDO.dir('bin/i18n/crowdin_credentials.yml'),
-    etags_json:             CDO.dir('bin/i18n/crowdin/etags/codeorg-testing_etags.json'),
-  },
-  'codeorg-markdown-testing': {
-    config_file:            CDO.dir('bin/i18n/crowdin/config/codeorg-markdown-testing_crowdin.yml'),
-    identity_file:          CDO.dir('bin/i18n/crowdin_credentials.yml'),
-    etags_json:             CDO.dir('bin/i18n/crowdin/etags/codeorg-markdown-testing_etags.json'),
-  },
-  'hour-of-code-test': {
-    config_file:            CDO.dir('bin/i18n/crowdin/config/hourofcode-testing_crowdin.yml'),
-    identity_file:          CDO.dir('bin/i18n/crowdin_credentials.yml'),
-    etags_json:             CDO.dir('bin/i18n/crowdin/etags/hour-of-code-testing_etags.json'),
-  },
-  'codeorg-restricted-test': {
-    config_file:            CDO.dir('bin/i18n/crowdin/config/codeorg-restricted-testing_crowdin.yml'),
-    identity_file:          CDO.dir('bin/i18n/crowdin_credentials.yml'),
-    etags_json:             CDO.dir('bin/i18n/crowdin/etags/codeorg-restricted-testing_etags.json'),
-  },
-}.freeze
-
 class I18nScriptUtils
+  CROWDIN_CREDS_PATH = CDO.dir('bin/i18n/crowdin_credentials.yml').freeze
   PROGRESS_BAR_FORMAT = '%t: |%B| %p% %a'.freeze
   PARALLEL_PROCESSES = Parallel.processor_count.freeze
   SOURCE_LOCALE = 'en-US'.freeze
   TTS_LOCALES = (::TextToSpeech::VOICES.keys - %i[en-US]).freeze
   TESTING_BY_DEFAULT = false
 
+  # @return [Hash] the Crowdin credentials.
+  #   @option crowdin_creds [String] 'api_token' the Crowdin API token.
+  def self.crowdin_creds
+    @crowdin_creds ||= YAML.load_file(CROWDIN_CREDS_PATH).freeze
+  end
```

Comment:
Would make sense to return a message if the path is not found?
something like:
"CROWDIN_CREDS_PATH not found. Please make sure a file containing a Crowdin "api_token" exists."

Reply:
1. If the file does not exist, the error will be raised: `No such file or directory @ rb_sysopen - /bin/i18n/crowdin_credentials.yml`
2. If no `api_token`, `Crowdin::Client` will raise an error.